packages/media/src/generateVideoStrip.js

```diff
@@ -112,6 +113,23 @@ async function generateVideoStrip(element, resource, stripWidth, stripHeight) {
     // Seek to the next offset
     await seekVideo(video, timeOffset);
 
+    const dx = frame * actualStripFrameWidth;
+
+    if (flip.vertical || flip.horizontal) {
+      // translate context to center of canvas
+      ctx.translate(
+        dx + actualStripFrameWidth / 2,
+        0 + actualStripFrameWidth / 2
+      );
+      // flip context
+      ctx.scale(flip.horizontal ? -1 : 1, flip.vertical ? -1 : 1);
+      // translate context back to orgin
+      ctx.translate(
+        -(dx + actualStripFrameWidth / 2),
+        -(0 + actualStripFrameWidth / 2)
+      );
```

Comment:
It is however a little annoying, that we have to do this after the `scale()`. I guess it could be done in a single operation using `setTransform()`, but this is more obvious, so works for me!

Reply:
Yes, I see what you mean. I did try using `ctx.setTransform()` initially but I couldn't get it to work consistently